### PR TITLE
Translate Release Channels

### PR DESCRIPTION
**Pull request by guilhermepiovesan**

### CODE REVIEW

**Comment by jhonmike** [comment]
@guilhermepiovesan o bot acabou enviando atualizações, consegue tratar os conflitos do seu PR que assim que possível faremos Code Review

**Comment by guilhermepiovesan** [comment]
@jhonmike obrigado por avisar, já fiz o rebase, está liberado pra review.

File: content/docs/release-channels.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+O grau de alteração entre o release mais recente do Next e a versão mais recente do Latest é aproximadamente o mesmo que você encontraria entre dois releases de minor semver. No entanto, **o canal Next não está de acordo com o controle de versão semântico.** Você deve esperar breaking changes ocasionais durante sucessivos releases no canal Next.
 
-**Do not use prereleases in user-facing applications.**
+**Não use prereleases em aplicações voltadas para o usuário final.**
 
-Releases in Next are published with the `next` tag on npm. Versions are generated from a hash of the build's contents, e.g. `0.0.0-1022ee0ec`.
+Releases no Next são publicadas com a tag `next` no npm. As versões são geradas a partir de um hash do conteúdo do build, por exemplo `0.0.0-1022ee0ec`.
 
-#### Using the Next Channel for Integration Testing {#using-the-next-channel-for-integration-testing}
+#### Usando o canal Next para Testes de Integração {#using-the-next-channel-for-integration-testing}
```
Comment (jhonmike) [inline content/docs/release-channels.md]:
```suggestion
#### Usando o Canal Next para Testes de Integração {#using-the-next-channel-for-integration-testing}
```

File: content/docs/release-channels.md [hunk 2]
```diff
@@ -60,36 +60,36 @@ If you're the author of a third party React framework, library, developer tool,
...
 
-Experimental releases may be significantly different than releases to Next and Latest. **Do not use Experimental releases in user-facing applications.** You should expect frequent breaking changes between releases in the Experimental channel.
+Os releases Experimental podem ser significativamente diferentes dos releases Next e Latest. **Não use releases Experimental em aplicações voltadas para o usuário final.** Você deve esperar breaking changes frequentes entre releases no canal Experimental.
 
-Releases in Experimental are published with the `experimental` tag on npm. Versions are generated from a hash of the build's contents, e.g. `0.0.0-experimental-1022ee0ec`.
+Releases no Experimental são publicados com a tag `experimental` no npm. As versões são geradas a partir de um hash do conteúdo do build, por exemplo `0.0.0-experimental-1022ee0ec`.
 
-#### What Goes Into an Experimental Release? {#what-goes-into-an-experimental-release}
+#### O que entra em um Release Experimental? {#what-goes-into-an-experimental-release}
```
Comment (jhonmike) [inline content/docs/release-channels.md]:
```suggestion
#### O Que Entra em um Release Experimental? {#what-goes-into-an-experimental-release}
```

File: content/docs/release-channels.md [hunk 3]
```diff
@@ -60,36 +60,36 @@ If you're the author of a third party React framework, library, developer tool,
...
 
-For example, if the Experimental channel had existed when we announced Hooks, we would have released Hooks to the Experimental channel weeks before they were available in Latest.
+Por exemplo, se o canal Experimental existisse quando anunciamos Hooks, teríamos lançado Hooks no canal Experimental semanas antes de estarem disponíveis no Latest.
 
-You may find it valuable to run integration tests against Experimental. This is up to you. However, be advised that Experimental is even less stable than Next. **We do not guarantee any stability between Experimental releases.**
+Você pode achar valioso executar testes de integração no Experimental. Isso é com você. No entanto, saiba que o Experimental é ainda menos estável que o Next. **Não garantimos estabilidade entre releases Experimental.**
 
-#### How Can I Learn More About Experimental Features? {#how-can-i-learn-more-about-experimental-features}
+#### Como posso aprender mais sobre os recursos Experimental? {#how-can-i-learn-more-about-experimental-features}
```
Comment (jhonmike) [inline content/docs/release-channels.md]:
```suggestion
#### Como Posso Aprender Mais Sobre os Recursos Experimental? {#how-can-i-learn-more-about-experimental-features}
```

File: content/docs/release-channels.md (hunk 1 above)
Comment (guilhermepiovesan) [reply content/docs/release-channels.md]: https://github.com/reactjs/pt-BR.reactjs.org/pull/367/commits/2085be2dbfbfb897b09d35a23acfbc05141c27cf

File: content/docs/release-channels.md (hunk 2 above)
Comment (guilhermepiovesan) [reply content/docs/release-channels.md]: https://github.com/reactjs/pt-BR.reactjs.org/pull/367/commits/2085be2dbfbfb897b09d35a23acfbc05141c27cf

File: content/docs/release-channels.md (hunk 3 above)
Comment (guilhermepiovesan) [reply content/docs/release-channels.md]: https://github.com/reactjs/pt-BR.reactjs.org/pull/367/commits/2085be2dbfbfb897b09d35a23acfbc05141c27cf

Comment (guilhermepiovesan) [reply content/docs/release-channels.md]: https://github.com/reactjs/pt-BR.reactjs.org/pull/367/commits/2085be2dbfbfb897b09d35a23acfbc05141c27cf

**Pull request merged**